static/app/components/feedback/widget/useFeedback.tsx

```diff
@@ -0,0 +1,36 @@
+import {useMemo} from 'react';
+import * as Sentry from '@sentry/react';
+
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+import useAsyncSDKIntegrationStore from 'sentry/views/app/asyncSDKIntegrationProvider';
+
+type FeedbackIntegration = NonNullable<ReturnType<typeof Sentry.getFeedback>>;
+
+export type UseFeedbackOptions = {
+  formTitle?: string;
+  messagePlaceholder?: string;
+};
+
+export function useFeedback({formTitle, messagePlaceholder}: UseFeedbackOptions) {
+  const config = useLegacyStore(ConfigStore);
+  const {state} = useAsyncSDKIntegrationStore();
+
+  // TODO(ryan953): remove the fallback `?? Sentry.getFeedback()` after
+  // getsentry is calling `store.add(feedback);`
+  const feedback =
+    (state.Feedback as FeedbackIntegration | undefined) ?? Sentry.getFeedback();
```

Comment:
This todo is to-done! Can drop the fallback now. https://github.com/getsentry/getsentry/pull/14390 was the one.